exo-go/src/process_helpers/process.go

```diff
@@ -0,0 +1,113 @@
+package processHelpers
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os/exec"
+	"strings"
+
+	"github.com/Originate/exosphere/exo-go/src/util"
+)
+
+// Process represents a exec.Cmd process
+type Process struct {
+	Cmd        *exec.Cmd
+	StdoutLog  func(string)
+	StdoutPipe io.ReadCloser
+	StdinPipe  io.WriteCloser
+	Output     string
+}
+
+// NewProcess is Process's constructor
+func NewProcess(commandWords ...string) *Process {
+	process := &Process{Cmd: exec.Command(commandWords[0], commandWords[1:]...)} //nolint gas
+	return process
+}
+
+// Log reads the stream from the given stdPipeReader, logs the
+// output, and update process.Output
+func (process *Process) Log(stdPipeReader io.Reader) {
```

Comment:
I think this method should be private as I don't expected anyone to call it directly. Its also only used internally in Start